### PR TITLE
=str #17759: Handle when request() throws in ActorInterpreter

### PR DESCRIPTION
**Pull request by drewhk**

### CODE REVIEW

**Comment by drewhk** [comment]
Fixes #17762

File: akka-stream/src/main/scala/akka/stream/impl/fusing/ActorInterpreter.scala [hunk 1]
```diff
@@ -145,7 +148,7 @@ private[akka] class BatchingActorInputBoundary(val size: Int, val name: String)
 
     case OnComplete                ⇒ onComplete()
     case OnError(cause)            ⇒ onError(cause)
```
Comment (ktoso) [inline akka-stream/src/main/scala/akka/stream/impl/fusing/ActorInterpreter.scala]: should then these be `tryOnError`, according to the same reasoning then?

Comment (drewhk) [inline akka-stream/src/main/scala/akka/stream/impl/fusing/ActorInterpreter.scala]: that's not the onError you are looking for ;)

Comment (drewhk) [inline akka-stream/src/main/scala/akka/stream/impl/fusing/ActorInterpreter.scala]: (line 127/130)

Comment (ktoso) [inline akka-stream/src/main/scala/akka/stream/impl/fusing/ActorInterpreter.scala]: lol, I see :-)

**Comment by ktoso** [comment]
LGTM

**Pull request merged**